### DISCUSSION TITLE
تمرین ها

### DISCUSSION THREAD

**Discussion by hayyaun** [hayyaun/kids#4] [General]
<div dir="rtl">

### ✅ گروه ارسال تمرین ها

برای ارسال تمرین ها اسکرول کنید پایین صفحه و پیام بدید.  
برای ارسال پیام باید اکانت داشته باشید.  

[![ثبت نام](https://img.shields.io/badge/Sign%20Up-6FDA44?style=for-the-badge&logo=github&logoColor=white)](https://github.com/join)

### 🚨 قوانین گروه 🚨  

- بعد از هر جلسه کد جدید رو دانلود کنید
- لطفا به هیچ عنوان عکس با گوشی نفرستید!
- فقط و فقط **اسکرین شات** از کدتون بصورت تمام صفحه بفرستید
- حتما و حتما **اسمتون** رو بصورت کامل زیر عکستون بنویسید

</div>

**Comment by Nikannejatifar** [comment]
![Screenshot 2024-11-28 235144](https://github.com/user-attachments/assets/d6d6cf90-d912-4d40-b22b-63ac1ba2dd6c)
نیکان نجاتی فر

**Reply by hayyaun** [reply]
.اینی که حل کردی شده ۳ دو اسمش. اگه خواستی میتونی تمرین ۳ یک رو هم حل کنی 
از اینی که حل کردی آسون تره عالیه که تونستی اینو حل کنی 🙌

**Comment by Mahdizk90** [comment]
![IMG_20241127_233508_192](https://github.com/user-attachments/assets/4d60916c-7409-49d5-9b0c-3d8ba38a7a66)
Exercises 1,2

**Comment by Nikannejatifar** [comment]
![Screenshot 2024-11-29 125724](https://github.com/user-attachments/assets/07492225-3d60-4e95-89de-8ec6f28672fb)

**Reply by hayyaun** [reply]
فراتر از عالی... 👌

**Comment by Mahdizk90** [comment]
![IMG_20241203_233316_833](https://github.com/user-attachments/assets/06b561af-a1f1-4996-8cc5-637bd575ba6f)
مهدی ذکائی

**Reply by hayyaun** [reply]
درود به شرفت... تمرین ۳.۲ یه کم سخت تره ولی چیزی نیست نتونی

**Comment by Mahdizk90** [comment]
![Untitled8](https://github.com/user-attachments/assets/46783f58-1d9e-4fab-ae02-3d117fad09d1)
تمرین ۳.۲ ذکائی

**Reply by hayyaun** [reply]
توی جواب هات ۴ هم نوشته شده که عدد اول نیست درسته؟
پس یه اشتباهی وجود داره... احتمالا مشکل از اینه که if آخر رو توی حلقه داخلی گذاشتی به جای بیرونی
در حد یه فاصله است ایرادت. درستش کن دوباره بفرست

**Comment by Mahdizk90** [comment]
![Untitled9](https://github.com/user-attachments/assets/63dd6a97-e2fd-4ec4-b99c-55415a110c93)
اینم ۳.۲ درست شده

**Reply by hayyaun** [reply]
درود بهت

**Comment by Mahdizk90** [comment]
![Untitled](https://github.com/user-attachments/assets/be1710ee-8154-4139-9c3f-b55009905d8b)
این درسته؟ برای ۳.۳

**Comment by Mahdizk90** [comment]
با کلی کمک ار helps های شما و کمی از گوگل برای اصطلاحات

**Reply by hayyaun** [reply]
حدودا فهمیدم قراره چیکار کنی.
لطفا با درسایی که خوندیم حلقه و اگه و ... اینا بنویسش
بعد که تموم شد اون کارو هم میتونی بکنی. تابع و آرایه و اینا
ایده استارت و اند هم عالی بود اونم اضافه کن اگه دوست داشتی

**Comment by Nikannejatifar** [comment]
![photo_2024-12-10_00-41-04](https://github.com/user-attachments/assets/303cdd8e-d2df-48b2-90a8-6851c77445e8)

**Reply by Nikannejatifar** [reply]
Nikan nejatifar

**Reply by Nikannejatifar** [reply]
exercise3-3

**Reply by hayyaun** [reply]
مختصر و مفید... و کاملا درست

**Comment by Mahdizk90** [comment]
![Untitled25](https://github.com/user-attachments/assets/561406cc-4b8d-4b9a-9da8-69dd8fd430c9)
Mahdi zokaei 
Is this correct?

**Reply by hayyaun** [reply]
Well done Mehdi!! It's all correct 💯 now you can receive inputs for start points (a , b) as you were planning before, to generate custom Fibonacci. (Please dm me the answer in telegram). Next we'll be defining a function if you'd like 😍

**Comment by Noyangolalizade** [comment]
![image](https://github.com/user-attachments/assets/b420a8a6-b508-4080-a8b1-c7c5d39b44d0)

**Reply by Noyangolalizade** [reply]
درست کار میکنه ولی نمیدونم چرا اعداد غیر 1 رو یجوری مینویسه

**Reply by hayyaun** [reply]
نه بنظرم کاملا درست انجام دادی! 🤩 آفرین بهت... فقط یه پرینت اضافه داخل حلقه داری که لازم نبوده... ولی درکل درسته!

**Comment by sepehradnrz282** [comment]
<img width="960" alt="Capture" src="https://github.com/user-attachments/assets/a978e43a-33b9-451b-a880-8177f8e6f372" />

3-1 Sepehrad

**Reply by hayyaun** [reply]
آفرین بهت! میتونی پرینت رو خارج از حلقه و بعدش بذاری که فقط یه بار جواب رو نشون بده!

**Comment by sepehradnrz282** [comment]
<img width="960" alt="Capture" src="https://github.com/user-attachments/assets/53151550-4469-4344-979d-abbf7d37484b" />

3-2

**Reply by hayyaun** [reply]
Sepehrad:
 فقط منم کاری که نویان واسه تمرین 3/1 رو انجام داد رو انجام دادمدادم ولی ارور داد

**Reply by hayyaun** [reply]
نه جوابت غلطه. لطفا از [راهنمای تمرین](https://github.com/hayyaun/kids/blob/main/helps/exercise-3-2.md) کمک بگیر.
احتمالا فایل هات هم قدیمی هستن که تمرین قدیمی رو داری حل میکنی. فایل های جدید رو دانلود کن.
کدی که زدی چون اشتباهات زیادی داره بیشتر گیج میشی سعی کن از اول بزنی
(میتونی از کد نیکان به جای نویان کمک بگیری)

**Comment by Noyangolalizade** [comment]
![Screenshot 2024-12-13 160840](https://github.com/user-attachments/assets/03ff4581-fdbe-4a8d-991b-0068308ad282)

**Reply by Noyangolalizade** [reply]
نویان گلعلی زاده تمرین 3-3 فیبوناچی

**Reply by hayyaun** [reply]
آفرین بهت 💯 خسنه نباشی

**Reply by Noyangolalizade** [reply]
اوکی بالاخره🤝🚀

**Comment by sepehradnrz282** [comment]
<img width="949" alt="coding" src="https://github.com/user-attachments/assets/1149055a-ad29-49fc-8b0f-3d45ddc06ff0" />
3-2

**Reply by hayyaun** [reply]
اوکی کار میکنه ولی نباید چند بار بنویسه ... درستش کن بفرست

**Comment by sepehradnrz282** [comment]
<img width="960" alt="coding 2" src="https://github.com/user-attachments/assets/651245b6-d292-41b5-9013-c41215433916" />
3-3

**Reply by hayyaun** [reply]
عالی 🥳 خسته نباشی

**Comment by Noyangolalizade** [comment]
![image](https://github.com/user-attachments/assets/f7660270-c846-4af5-adb8-b8e5bcb6be30)

**Reply by Noyangolalizade** [reply]
noyan golalizade 3-3 after debug and fixing for better view

**Reply by hayyaun** [reply]
خب این که الان غلطه جوابش ... ۶ عدد اوله؟؟

**Comment by Sadrazero** [comment]
![image](https://github.com/user-attachments/assets/22893aa8-093a-46b5-8d73-da80b4e8d94e)
I did this sadra ghasempoor

**Reply by Sadrazero** [reply]
![Uploading image.png…]()

**Reply by hayyaun** [reply]
سلام صدرا جان نموندی آپلود کامل شه قبل ارسال پیام.
لطفا بعد آپلود فایل چند لحظه بمون تا کامل آپلود شه بعد دکمه ارسال رو بزن ❤️🙏

**Comment by Nikannejatifar** [comment]
![image](https://github.com/user-attachments/assets/5d64d95f-9e2d-4e2a-a43b-533752c54336)

**Reply by hayyaun** [reply]
اشتباه! الان بجای ۱۰ بگه سوال صدتا میخای دستی همشو بنویسی؟

**Comment by Noyangolalizade** [comment]
![IMG_20250130_133601_656](https://github.com/user-attachments/assets/4f63456d-d504-4df6-ba44-3af98ba43403)

**Reply by Noyangolalizade** [reply]
4.2 noyan
4.1 is in progress (editing some codes)

**Reply by hayyaun** [reply]
عالیه خسته نباشی. میتونی برای تمرین اون قسمت گرفتن متن رو داخل حلقه بذاری تا این کار هی تکرار شه.

**Comment by SAFACITY2001** [comment]
file:///C:/Users/ali/Pictures/Screenshots/Screenshot%20(58).png
تمرین ۲

**Reply by hayyaun** [reply]
عکس آپلود نشده...

**Comment by Nikannejatifar** [comment]
![image](https://github.com/user-attachments/assets/57933396-4a57-4894-9a1c-f9158351b586)

**Reply by Nikannejatifar** [reply]
4-2 exercise

**Reply by hayyaun** [reply]
زیبا تر ازین نمیشد نوشت. آفرین! 🤩

**Comment by mahdizk123** [comment]
![20250205_220500](https://github.com/user-attachments/assets/ccdd085e-bc15-486e-87f4-8a2395e42a91)

**Reply by hayyaun** [reply]
درسته کدت مشکلی نداره...
مهدی جان قوانین گروه رو نخوندی؟

**Reply by hayyaun** [reply]
https://github.com/hayyaun/kids/discussions/4#discussion-7554993

**Comment by Noyangolalizade** [comment]


**Reply by Noyangolalizade** [reply]
5.1 noyan golalizade

**Reply by hayyaun** [reply]
جواب ترمینال باید سه خط باشه، الان None نشون میده که یعنی خالی!!
۹۰ درصد کدت درسته فقط از کلیدواژه return استفاده نکردی
مشکل رو حل کن دوباره بفرست
